src/main/java/indexer/Index.java

```diff
@@ -11,24 +11,28 @@
 import java.util.stream.Collectors;
 
 public class Index {
-    public static String docs_file_path = "documents/";
-    public static String output_dir = "indicies/doc-index.tsv";
-    public static String docSize = "indicies/doc-size-index.tsv";
-    public static String tokenPositionsFile = "indicies/token-pos.tsv";
+    public static String DOC_DIRECTORY = "documents/";
```

Comment:
should these constants be final? Should we change the naming scheme to not be like python?